orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/ScmService.java

```diff
@@ -0,0 +1,32 @@
+package com.netflix.spinnaker.orca.igor;
+
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Nullable;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class ScmService {
+  public List compareCommits(
+      String repoType,
+      String projectKey,
+      String repositorySlug,
```

Comment:
Inconsistent param names - here its repositorySlug, above its repository

Reply:
This is an example of what I was saying in my previous comment. This API was moved from `BuildService`, I didn't actually write it, and it used the Atlassian-ish names. Let's see what we agree on, then I'll change one or the other accordingly.